api/src/services/tus/data-store.ts

```diff
@@ -14,7 +14,6 @@ export type TusDataStoreConfig = {
 	constants: {
 		ENABLED: boolean;
 		CHUNK_SIZE: number | null;
-		MAX_SIZE: number | null;
```

Comment:
Not used in the store. Is validated at server level...
Should we keep this or can it be removed for simplification?